packages/keystone/index.js

```diff
@@ -1,14 +1,10 @@
 const Keystone = require('./lib/Keystone');
 const PasswordAuthStrategy = require('./lib/auth/Password');
-const TwitterAuthStrategy = require('./lib/auth/Twitter');
-const { FacebookAuthStrategy } = require('./lib/auth/Facebook');
 const { BaseKeystoneAdapter, BaseListAdapter, BaseFieldAdapter } = require('./lib/adapters');
 
 module.exports = {
   Keystone,
   PasswordAuthStrategy,
-  TwitterAuthStrategy,
-  FacebookAuthStrategy,
   BaseKeystoneAdapter,
```

Comment:
Because we are removing these from the `@keystone-alpha/keystone` package we will need to add a `changeset` with a `major` version bump.

Reply:
consolidating this with previous one in a new `changeset`